content/bn/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,136 @@
+
+ব্যাকগ্রাউন্ড ক্যাসকেডিং ডিলিশনে, Kubernetes API সার্ভার অবজেক্টটি অবিলম্বে মুছে ফেলে এবং কন্ট্রোলার ব্যাকগ্রাউন্ডে নির্ভরশীল অবজেক্টগুলি পরিষ্কার করে। স্বাভাবিকভাবে, Kubernetes ব্যাকগ্রাউন্ড ক্যাসকেডিং ডিলিশন ব্যবহার করে যতক্ষণ না আপনি ম্যানুয়ালি ফরগ্রাউন্ড ডিলিশন ব্যবহার করেন বা নির্ভরশীল অবজেক্টগুলি অরফান না করেন।
+
+আরো জানার জন্য [ব্যাকগ্রাউন্ড ক্যাসকেডিং ডিলিশন ব্যবহার করুন।](/bn/docs/tasks/administer-cluster/use-cascading-deletion/#use-background-cascading-deletion)
+to learn more.
+
+### অরফানড ডিপেনডেন্টস  {#orphaned-dependents}
+
+যখন কুবার্নেটিস একটি মালিক অবজেক্ট মুছে ফেলে, তখন বাকি থাকা ডিপেনডেন্টসগুলোকে অরফান অবজেক্ট বলা হয়। ডিফল্টভাবে, কুবার্নেটিস ডিপেনডেন্ট অবজেক্টগুলো মুছে ফেলে। এই আচরণটি কিভাবে ওভাররাইড করবেন তা জানতে, দেখুন [মালিক অবজেক্ট মুছে ফেলুন এবং অরফান ডিপেনডেন্টস](/bn/docs/tasks/administer-cluster/use-cascading-deletion/#set-orphan-deletion-policy)
+
+## অব্যবহৃত কনটেইনার এবং চিত্রগুলির গার্বেজ কালেকশন {#containers-images}
+
+{{<glossary_tooltip text="kubelet" term_id="kubelet">}} প্রতি দুই মিনিটে অব্যবহৃত চিত্রগুলির গার্বেজ কালেকশন এবং প্রতি মিনিটে অব্যবহৃত কনটেইনারগুলির গার্বেজ কালেকশন করে। বাহ্যিক গার্বেজ কালেকশন টুলস ব্যবহার থেকে বিরত থাকুন, কারণ এটি কিউবিলেটের আচরণ ভেঙে ফেলতে পারে এবং এমন কনটেইনারগুলি মুছে ফেলতে পারে যা থাকার উচিত।
+অব্যবহৃত কনটেইনার এবং চিত্রগুলির গার্বেজ কালেকশন জন্য বিকল্প কনফিগার করতে, কিউবিলেটটি একটি [কনফিগারেশন ফাইল](/bn/docs/tasks/administer-cluster/kubelet-config-file/) ব্যবহার করে কনফিগার করুন এবং গার্বেজ কালেকশন সম্পর্কিত প্যারামিটারগুলি পরিবর্তন করুন [`KubeletConfiguration`](/bn/docs/reference/config-api/kubelet-config.v1beta1/)
+রিসোর্সটি ব্যবহার করে।
+
+### কনটেইনার চিত্রের জীবনচক্র {#container-image-lifecycle}
+
+কুবেরনেটিস তার *চিত্র ম্যানেজার（Image Manager)* এর মাধ্যমে সমস্ত চিত্রের জীবনচক্র পরিচালনা করে, যা কিউবিলেটের অংশ এবং {{< glossary_tooltip text="cadvisor" term_id="cadvisor" >}}এর সহযোগিতায়. Kubelet
+গার্বেজ কালেকশন সিদ্ধান্ত নেবার সময় নিম্নলিখিত ডিস্ক ব্যবহার সীমা বিবেচনা করে:
+
+* `HighThresholdPercent`
+* `LowThresholdPercent`
+
+ডিস্ক ব্যবহারের পরিমাণ নির্ধারিত `HighThresholdPercent` মানের চেয়ে বেশি হলে গার্বেজ কালেকশন ট্রিগার হয়, যা চিত্র (ইমেজ) মুছে ফেলে। এটি সর্বশেষ ব্যবহারের সময়ের ভিত্তিতে প্রাচীনতম চিত্র থেকে শুরু করে মুছে ফেলা হয়। Kubelet চিত্র মুছতে থাকে যতক্ষণ না ডিস্ক ব্যবহারের পরিমাণ `LowThresholdPercent` মানে নেমে আসে।
+
+#### অব্যবহৃত কন্টেইনার ইমেজগুলোর জন্য গারবেজ কালেকশন {#image-maximum-age-gc}
+
+{{< feature-state feature_gate_name="ImageMaximumGCAge" >}}
+
+একটি বিটা ফিচার হিসেবে, আপনি নির্দিষ্ট করতে পারেন যে একটি স্থানীয় ইমেজ সর্বাধিক কতক্ষণ অব্যবহৃত থাকতে পারবে, ডিস্ক ব্যবহারের পরিমাণ নির্বিশেষে। এটি একটি  kubelet সেটিং, যা প্রতিটি নোডের জন্য কনফিগার করা যায়।
+
+এই সেটিং কনফিগার করতে, কুবলেটের জন্য `ImageMaximumGCAge`
+[ফিচার গেট](/bn/docs/reference/command-line-tools-reference/feature-gates/) সক্ষম করুন এবং kubelet কনফিগারেশন ফাইলে `imageMaximumGCAge` ফিল্ডের জন্য একটি মান সেট করুন।
```

Comment:
Use the English version as reference `[ফিচার গেট](/docs/reference/command-line-tools-reference/feature-gates/)`